### DISCUSSION TITLE
need sound

### DISCUSSION THREAD

**Discussion by SuQiANQAQ** [MonitorControl/MonitorControl#1107] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

I have not sound

### Describe the solution you'd like

如果有虚拟声卡代替系统音乐就可以了。类似的app，speakamp。我现在控制亮度用的moniter app，控制声音的speakamp，比较麻烦。

### Describe alternatives you've considered

然后虚拟声卡可以选择播放的声音，并且记忆不同设备的默认声音，不然切成耳机声音突然变大会聋。

### Anything else?

no

**Comment by waydabber** [comment]
Translation of the issue:

**Describe the solution you'd like**

If there is a virtual sound card to replace the system music, it will be fine. Similar app, speakamp. The monitor app I use to control the brightness and the speakamp of the sound are more troublesome.

**Describe alternatives you've considered**

Then the virtual sound card can choose the sound to play, and memorize the default sound of different devices, otherwise the sound of the earphone will suddenly become deaf.

**Comment by waydabber** [comment]
I am not sure I fully understand the issue. However MonitorControl specifically deals with controlling a display's built-in amplifier via DDC commands and is not concerned with general volume control for macOS. I understand that it would be neat to have an all-encompassing solution regarding this, but it simply is out of scope for this app.
